src/components/voltage-init-result.js

```diff
@@ -199,6 +209,59 @@ const VoltageInitResult = ({ result, status, tabIndex, setTabIndex }) => {
         ];
     }, []);
 
+    function renderTotalReactiveSlacks(reactiveSlacks) {
+        const calculateTotal = (reactiveSlacks, isPositive) => {
+            return reactiveSlacks
+                ? reactiveSlacks
+                      .filter((reactiveSlack) =>
+                          isPositive
+                              ? reactiveSlack.slack > 0
+                              : reactiveSlack.slack < 0
+                      )
+                      .reduce(
+                          (sum, reactiveSlack) => sum + reactiveSlack.slack,
+                          0
+                      )
+                : 0;
+        };
+
+        const totalInjection = calculateTotal(
+            resultToShow.reactiveSlacks,
```

Comment:
you have to pass the function param "reactiveSlacks" here instead of "resultToShow.reactiveSlacks".
otherwise the function param is useless.